automated install/basic-install.sh

```diff
@@ -1884,29 +1875,23 @@ get_binary_name() {
         local rev
         rev=$(echo "${cpu_arch}" | grep -o '[0-9]*')
         if [[ "${machine}" == "aarch64" ]]; then
```

Comment:
The whole `aarch64` block can be taken out from the `if [[ "${machine}" == "arm"* || "${machine}" == *"aarch"* ]];` block as there is a straight path to it and no down-stream decisions need to be made. It can be on the same level as `x86_64` or `riscv64`.